Mathlib/Algebra/Field/Power.lean

```diff
@@ -17,6 +17,12 @@ beyond those needed for the basic definition.
 
 variable {α : Type*}
 
+/-- a⁻¹ * d ^ s * a = (a⁻¹ * d * a) ^ s when a is not zero -/
+lemma conj_nonComm_Algebra [DivisionSemiring α] (s : ℕ) (a d : α) (ha : a ≠ 0) :
```

Comment:
Can you try making both `s` and `a,d` implicit (so in curly brackets `{` and `}` rather than in usual ones `(` and `)`)?